onfido/models.py

```diff
@@ -537,5 +537,5 @@ def parse(self, raw_json: dict) -> Event:
         obj = payload["object"]
         self.onfido_id = obj["id"]
         self.status = obj["status"]
-        self.completed_at = date_parse(obj["completed_at"])
+        self.completed_at = date_parse(obj["completed_at_iso8601"])
```

Comment:
The other change (not sure if this is breaking - having problems testing callback).